app/konnect/updates.md

```diff
@@ -11,12 +11,21 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## November 2022
 
+
+### 2022.11.xx
+
+**New Analytics predefined teams**
+: {{site.konnect_short_name}} now allows you to add users to the Analytics Viewer and Analytics Admin teams. These teams allow you to give users access to only the Analytics section in {{site.konnect_short_name}}. The Analytics Viewer team only allows users to view and export analytics. The Analytics Admin team allows users to view, create, edit, and delete all analytics. 
```

Comment:
@cloudjumpercat How do you feel adding a "so what" into the release notes. Something like:

> With this release, individual users do not have to be Org Admins anymore to access all the Analytics capabilities.